yt_dlp/extractor/zoom.py

```diff
@@ -51,27 +47,48 @@ def _real_extract(self, url):
                 raise ExtractorError(validation['errorMessage'], expected=True)
             webpage = self._download_webpage(url, play_id)
 
+        data: dict[str, Union[None, str, int, list, bool, dict[str, Any]]]
         data = self._parse_json(self._search_regex(
             r'(?s)window\.__data__\s*=\s*({.+?});',
             webpage, 'data'), play_id, js_to_json)
 
         subtitles = {}
-        for _type in ('transcript', 'cc'):
+        for _type in ('transcript', 'cc', 'chapter'):
             if data.get('%sUrl' % _type):
                 subtitles[_type] = [{
                     'url': urljoin(base_url, data['%sUrl' % _type]),
                     'ext': 'vtt',
                 }]
 
+        formats: list[dict[str, Union[None, str, int]]] = []
+
+        if data.get('shareMp4Url'):
+            formats.append({
+                'format_note': 'Screen share stream',
+                'url': str_or_none(data.get('shareMp4Url')),
+                'width': int_or_none(data.get('shareResolvtionsWidth')),
+                'height': int_or_none(data.get('shareResolvtionsHeight')),
+                'format_id': str_or_none(data.get('shareVideoId')),
+                'ext': 'mp4',
+            })
+
+        formats.append({
+            'format_note': 'Camera stream',
+            'url': str_or_none(data.get('viewMp4Url')),
+            'width': int_or_none(data.get('viewResolvtionsWidth')),
+            'height': int_or_none(data.get('viewResolvtionsHeight')),
+            'format_id': str_or_none(data.get('recordingId')),
+            'ext': 'mp4',
+            'filesize_approx': parse_filesize(data.get('fileSize')),
+        })
+
         return {
```

Comment:
`self._sort_formats` before returning. Add `preference` to the dicts (with comment explaining why) if you want to maintain a specific order